src/Hazelcast.Net/Protocol/BuiltInCodecs/EntryListIntegerGuidCodec.cs

```diff
@@ -21,7 +21,7 @@ namespace Hazelcast.Protocol.BuiltInCodecs
 {
     internal static class EntryListIntegerUUIDCodec
     {
-        private const int EntrySizeInBytes = BytesExtensions.SizeOfInt + BytesExtensions.SizeOfGuid;
+        private const int EntrySizeInBytes = BytesExtensions.SizeOfInt + BytesExtensions.SizeOfCodecGuid;
```

Comment:
Does this change apply to codec template? Or should we check codecs each time after regenerated?

Reply:
The codec templates have been updated. So there should be a protocol repo PR too. You'll need to have a look at [this PR](https://github.com/hazelcast/hazelcast-client-protocol/pull/442) and ideally review/approve too.